pystiche_papers/ulyanov_et_al_2016/_data.py

```diff
@@ -1,18 +1,19 @@
-from typing import List, Optional, Sized
+from typing import List, Optional
 from urllib.parse import urljoin
+import itertools
 
-from torch.utils.data import DataLoader, Dataset, Sampler
+import torch.utils.data
+from torch.utils.data import DataLoader, Dataset
```

Comment:
Why both? Either import everything you need by name or only import the namespace and access everything from there.